### ISSUE TITLE
Key press and release not working in Debain

### ISSUE THREAD

**Issue by nodminger** [TolikPylypchuk/SharpHook#64]
`KeyPress` and `KeyRelease` is not working on Debian for any keys(a,b,c....SHIFT,Backspace,CTRL,.,etc). I also tried it with Delay(`Thread.Sleep(50)`). All other functions like mouse press, release and simulating text works perfectly.

My Debian info: Linux Debian 5.10.0-24-amd64 #1 SMP Debian 5.10.179-5 (2023-08-08) x86_64 GNU/Linux.

```
using SharpHook;
using SharpHook.Native;

class Program
{
    static void Main()
    {
        var simulator = new EventSimulator();
        simulator.SimulateKeyPress(KeyCode.VcBackspace);
        simulator.SimulateKeyRelease(KeyCode.VcBackspace);
    }
}
```

**Comment by TolikPylypchuk** [comment]
Hi! Thanks for posting this issue! I'll check it out shortly and let you know more about it.

**Comment by ZeoWorks** [comment]
+1, keypress/release does not work on Ubuntu and other linux variants.

**Comment by TolikPylypchuk** [comment]
Could you please check if version 4.2.1 works correctly on Linux? I haven't had time yet to look into this issue.

**Comment by ZeoWorks** [comment]
Hi, I can confirm 4.2.1 works fine, the issue only exists with 5.0.0.

**Comment by ZeoWorks** [comment]
However, SharpHook.Native.KeyCode.VcUp & other arrow keys seem to return the incorrect button input.

**Comment by TolikPylypchuk** [comment]
Hi @nodminger and @ZeoWorks! I've just tried to reproduce this issue on a Debian 12 VM, and everything works perfectly. Could you please share the [libuiohook debug logs](https://sharphook.tolik.io/v5.0.0/articles/logging.html) so that we have more info?

**Comment by nodminger** [comment]
@TolikPylypchuk 
I tried to get logs using this code, but I got errors in the code. I think I am using the logging code in a wrong way.
```
using SharpHook;
using SharpHook.Native;
using SharpHook.Logging;

class Program
{
    static async Task Main()
    {
        Thread.Sleep(3000);
        var logSource = LogSource.Register(minLevel: LogLevel.Info);
        logSource.MessageLogged += this.OnMessageLogged;

        var simulator = new EventSimulator();

        simulator.SimulateKeyPress(KeyCode.VcC);

        simulator.SimulateKeyRelease(KeyCode.VcC);
    }

    private void OnMessageLogged(object? sender, LogEventArgs e)
    {
        this.logger.Log(this.AdaptLogLevel(e.LogEntry.Level), e.LogEntry.FullText);
    }
}
```

**Errors**
**Line 11:** Keyword 'this' is not valid in a static property, static method, or static field initializer
**Line 22:** 'Program' does not contain a definition for 'logger' and no accessible extension method 'logger' accepting a first argument of type 'Program' could be found (are you missing a using directive or an assembly reference?)
**Line 22:** 'Program' does not contain a definition for 'AdaptLogLevel' and no accessible extension method 'AdaptLogLevel' accepting a first argument of type 'Program' could be found (are you missing a using directive or an assembly reference?)

**Comment by TolikPylypchuk** [comment]
Well, yes, you shouldn't copy the example code verbatim 😄 

You should do this instead:

```c#
private static void OnMessageLogged(object? sender, LogEventArgs e)
{
    Console.WriteLine($"{e.LogEntry.Level}: {e.LogEntry.FullText}");
}
```

And then when you add this method to the event, drop `this`:

```c#
logSource.MessageLogged += OnMessageLogged;
```

**Comment by nodminger** [comment]
@TolikPylypchuk 
Thanks for the example. I changed my code to
```
using SharpHook;
using SharpHook.Native;
using SharpHook.Logging;

class Program
{
    static async Task Main()
    {
        Thread.Sleep(3000);

        var simulator = new EventSimulator();
        var logSource = LogSource.Register(minLevel: LogLevel.Info);
        logSource.MessageLogged += OnMessageLogged;

        simulator.SimulateKeyPress(KeyCode.VcC);
        simulator.SimulateKeyRelease(KeyCode.VcC);
    }

    private static void OnMessageLogged(object? sender, LogEventArgs e)
    {
        Console.WriteLine($"{e.LogEntry.Level}: {e.LogEntry.FullText}");
    }
}
```
and got this output
```
Warn: post_key_event [45]: Unable to lookup scancode: 67
Warn: post_key_event [45]: Unable to lookup scancode: 67
```

**Comment by TolikPylypchuk** [comment]
Ok, I know what the issue is, and I will release version 5.1.1 with a fix in a couple days.

**Comment by TolikPylypchuk** [comment]
I've just released version 5.1.1 which fixes this problem, so the issue can be closed.
